doc/source/release/1.16.0-notes.rst

```diff
@@ -127,8 +153,18 @@ Array API Standard Support
 Experimental support for array libraries other than NumPy has been added to
 existing sub-packages in recent versions of SciPy. Please consider testing
 these features by setting an environment variable ``SCIPY_ARRAY_API=1`` and
-providing PyTorch, JAX, or CuPy arrays as array arguments.
-Features with support added for SciPy 1.16.0 include:
+providing PyTorch, JAX, or CuPy arrays as array arguments. Many functions
+in `scipy.stats`, `scipy.special`, `scipy.optimize`, and `scipy.constants`
+now provide tables documenting compatible array and device types as well as
+support for lazy arrays and JIT compilation. New features with support and
+old features with support added for SciPy 1.16.0 include:
+
+- Most features of `scipy.signal`
```

Comment:
```suggestion
- Most features of `scipy.cluster`
- Most features of `scipy.signal`
```

Reply:
Note: this is regardless of https://github.com/scipy/scipy/pull/22960

Reply:
Those were added before this release, though.

Reply:
None of `cluster` worked with `jax.jit` in 1.15. Dask was not even a thing.

I'm changing `old features with support added` to `old features with support added or enhanced`.

Reply:
I didn't include new backends here. Then, as you say, essentially everything we have had in the past would appear on this list again. I'm just including functions that are getting translated for the first time.